app/helpers/application_helper.rb

```diff
@@ -53,4 +53,49 @@ def render_index_grid(grid)
     no_records = grid.current_page_records.length < 1
     render "admin/common/grid", no_records: no_records, grid: grid
   end
+
```

Comment:
I'm fine with just a separate helper in this case since the code will eventually go away anyway.